ui/raidboss/data/05-shb/raid/e9n.js

```diff
@@ -1,8 +1,153 @@
+import Conditions from '../../../../../resources/conditions.js';
+import NetRegexes from '../../../../../resources/netregexes.js';
+import { Responses } from '../../../../../resources/responses.js';
 import ZoneId from '../../../../../resources/zone_id.js';
 
 export default {
   zoneId: ZoneId.EdensPromiseUmbra,
   timelineFile: 'e9n.txt',
   triggers: [
+    {
+      id: 'E9N Ground-Razing Particle Beam',
+      netRegex: NetRegexes.startsUsing({ id: '55ED', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E9N The Art Of Darkness Right',
+      netRegex: NetRegexes.startsUsing({ id: '5223', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 4,
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'E9N The Art Of Darkness Left',
+      netRegex: NetRegexes.startsUsing({ id: '5224', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 4,
+      response: Responses.goRight(),
+    },
+    {
+      id: 'E9N Wide-Angle Particle Beam',
+      netRegex: NetRegexes.startsUsing({ id: '5AFF', source: 'Cloud Of Darkness', capture: false }),
+      response: Responses.awayFromFront('info'),
+    },
+    {
+      id: 'E9N Zero-Form Particle Beam',
+      netRegex: NetRegexes.startsUsing({ id: '55EB', source: 'Cloud Of Darkness' }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'E9N Empty Plane',
+      netRegex: NetRegexes.startsUsing({ id: '4FC6', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E9N Mire Of Despair',
+      netRegex: NetRegexes.startsUsing({ id: '5B07', source: 'Cloud Of Darkness', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Out from Under',
+        },
+      },
+    },
+    {
+      id: 'E9N Obscure Woods',
+      netRegex: NetRegexes.startsUsing({ id: '4FA2', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E9N Waste Away',
+      netRegex: NetRegexes.startsUsing({ id: '55DE', source: 'Cloud Of Darkness', capture: false }),
+      response: Responses.knockback('info'),
+    },
+    {
+      id: 'E9N Stygian Tether',
+      netRegex: NetRegexes.headMarker({ id: '000C' }),
+      condition: Conditions.targetIsYou(),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Wait Near Bramble',
```

Comment:
Sorry, can you remind me again what this is too?

Reply:
This is the "wait for your harmless tether to turn into a thorn tether" bit.

Reply:
Yea, you can hang out near your thorn patch with the tether, you also have a purple floor marker beneath you. Once that floor marker goes away you're safe to run away. Only the distance you run once the floor marker disappears is what counts, so this would be so that you don't run across the map, and then run out of room for running / drag the boss across the map.

Reply:
Ahaha thanks for the mechanic explanation.  I didn't actually realize it's how it worked.